apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -685,6 +693,18 @@ function getCloseButtonUrl( calypsoPort ) {
 
 		window.wp.hooks.doAction( 'updateCloseButtonOverrides', data );
 	};
+
+	addFilter(
+		'a8c.WpcomBlockEditorNavSidebar.closeUrl',
+		'wpcom-block-editor/getCloseButtonUrl',
+		( closeUrl ) => calypsoifyGutenberg.closeUrl || closeUrl
+	);
+
+	addFilter(
+		'a8c.WpcomBlockEditorNavSidebar.closeLabel',
```

Comment:
Another tiny nit: the second part of this filter name uses Pascal case, where the `wpcom-block-editor` is kebab case. One is the name at a package level, whereas the other is a component within a package?

Reply:
Yeah the naming is interesting. Most of the examples I found for hooks/filters in Gutenberg were using component names e.g. [`editor.PostPreview.interstitialMarkup`](https://github.com/WordPress/gutenberg/blob/ac22829574d187b96773afb5e95eeda67eb69470/packages/editor/src/components/post-preview-button/index.js#L96). Maybe the prefix in that case should be the plugin name like `full-site-editing.*`? But a8c seems more unique than that.

Reply:
Yep, `a8c` is nice and unique for our purposes. I'm not too fussed personally, and it's clear enough to us that the the Pascal case refers to a particular component. We can always rename in the future if another naming scheme becomes clear to us!